NEWS.md

```diff
@@ -6,8 +6,7 @@
 
 ## Bug fixes
 
-* `inner_combine_linter()` no longer throws on length-1 calls to `c()` like `c(exp(2))` or `c(log(3))` (#2017, @MichaelChirico). Such usage is discouraged by `unnecessary_concatenation_linter()`, but `inner_combine_linter()` _per se_ does not apply.
```

Comment:
Why move these bugfixes?

Reply:
they're not really bugfixes (which for us is like the issue at hand, where running lint() throws an error) but lint precision/recall fixes. those have gone under new/improved for us historically (and in the ongoing release)